src/main/resources/config/application-dev.yml

```diff
@@ -47,7 +47,7 @@ spring:
   cloud:
     gateway:
       default-filters:
-        - SetAuthorizationHeader
+        - JWTRelay
```

Comment:
this is only for dev profile ? What happened in prod profile ?

Reply:
Maybe we can put this in application.yml because it's the same in prod profile ?

Reply:
sure !